### ISSUE TITLE
A11y keyboard control Tab component TabList TabPanel focus issue

### ISSUE THREAD

**Issue by DaveBest99** [angular/components#3423]
#### Bug, feature request, or proposal:
Bug: Screen reader TabPanel focus issue.

#### What is the expected behavior?
Pressing the Tab key from the TabList should move the focus to the first focusable element in the TabPanel displaying the associated Tab element content. Pressing Shift+Tab key from the TabList should move the focus to the previous page element. The screen reader will read the element text label and show it on the braille display.

#### What is the current behavior?
Currently the TabPanel has no focusable element and the screen reader focus is taken past the Tab component.

#### Which versions of Angular, Material, OS, browsers are affected?
Testing was performed on Windows10 with JAWS17 and Firefox/IE11/Chrome browsers.

#### Is there anything else we should know?
The TabPanel has no focusable element at the top. If the TabPanel has no focusable elements then the user will be taken to the next page element after the Tab component.
For example, on the Angular Material components page, selecting the "API Reference" and then pressing Tab key moves focus to the "Learn Angular" link in the page footer.
